Shared/Samples/Run valve isolation trace/RunValveIsolationTraceView.swift

```diff
@@ -0,0 +1,111 @@
+// Copyright 2023 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+
+struct RunValveIsolationTraceView: View {
+    /// The view model for the sample.
+    @StateObject var model = Model()
+    
+    var body: some View {
+        MapViewReader { mapViewProxy in
+            MapView(
+                map: model.map,
+                graphicsOverlays: [model.parametersOverlay]
+            )
+            .onSingleTapGesture { screenPoint, mapPoint in
+                model.lastSingleTap = (screenPoint, mapPoint)
+            }
+            .overlay(alignment: .top) {
+                Text(model.statusText)
+                    .frame(maxWidth: .infinity, alignment: .center)
+                    .padding(8)
+                    .background(.ultraThinMaterial, ignoresSafeAreaEdges: .horizontal)
+                    .multilineTextAlignment(.center)
+            }
+            .task {
+                try? await model.setup()
+                await mapViewProxy.setViewpointCenter(model.startingLocationPoint, scale: 3_000)
```

Comment:
why not just set an initial viewpoint on the map in the model init?

map.initialViewpoint = ...